### PR TITLE
🎨 Website header #1

### PR DESCRIPTION
**Pull request by jelizasilatei**
# Description

This pull request introduces a new website header/navbar, enhancing the user experience and navigation. 
The header includes a logo, navigation links, an action button, and a responsive dropdown menu for smaller screens. 
It has been meticulously designed and styled to ensure a seamless and visually appealing transition across various screen sizes. 

Fixes #1

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Desktop:
Desktop- website header

Tablet:
Tablet(toggle-bars) Website header
Tablet(dropdown-menu) Website-header

Mobile:
Mobile(togle-bar) Website header
Mobile(dropdown-menu) Website header

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jelizasilatei** [comment]
I worked on this with Morris Gitonga: https://github.com/Lord-Gitonga.
Kindly review my pull request.

File: src/css/style.css
Comment (paulkahura) [inline src/css/style.css:1]: Great job! One minor change: the font should be the one in the designs.

**Review by paulkahura — CHANGES_REQUESTED** [review]

File: src/css/style.css
Comment (jelizasilatei) [reply src/css/style.css:1]: Thank you,
I have incorporated the fonts required.

**Comment by JAPHETHNYARANGA** [comment]
closing as this is done, please pick another issue

**Pull request closed without merging**